hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/cache/CacheManager.java

```diff
@@ -60,18 +60,18 @@
     private static final String RESET_PUBLISH_CACHE_PROP = "hawkular-alerts.reset-publish-cache";
     private static final String RESET_PUBLISH_CACHE_ENV = "RESET_PUBLISH_CACHE";
 
-    private volatile boolean updateRequested = false;
-    private volatile boolean updating = false;
-
     @EJB
     PropertiesService properties;
 
     @EJB
     DefinitionsService definitions;
 
+    @Resource(lookup = "java:jboss/infinispan/cache/hawkular-alerts/dataIds")
```

Comment:
+1
Additional comments will help to follow better the design.
I'm ok with publishDataIdsCache suggestion.
I will update it.